src/applications/simple_storage_starknetjs.md

```diff
@@ -0,0 +1,259 @@
+# Simple Storage (Starknet-js + Cairo)
+
+In this tutorial, you will write and deploy a SimpleStorage Cairo contract on Starknet Sepolia Testnet. Then, you will learn how you can interact with the contract using Starknet-js. 
+
+In order to fully understand and implement the content of this tutorial, here are the prerequisites:
+- Basic knowledge of Cairo: you can get familiar with the Cairo language, and learn how to build a simple smart contract with Cairo [here](https://book.cairo-lang.org/).
+
+- Scarb for compiling Cairo code and packaging support: follow [here](https://docs.swmansion.com/scarb/download.html).
+
+- Starkli for the declaration and deployment of Cairo contracts: follow [here](https://book.starkli.rs/installation).
+
+## Writing SimpleStorage contract in Cairo
+The SimpleStorage contract has only one purpose: storing a number. We want the users to interact with the stored number by **viewing** the current stored number and **setting** a new number.
```

Comment:
Maybe the word _reading_ and _writing_ is more explicit that _viewing_ and _setting_